robusta_krr/main.py

```diff
@@ -271,7 +271,8 @@ def run_strategy(
                     logger.exception("Error occured while parsing arguments")
                 else:
                     runner = Runner()
-                    asyncio.run(runner.run())
+                    exit_code = asyncio.run(runner.run())
+                    raise typer.Exit(code=exit_code)
```

Comment:
what is the benifits of using typer.exit? (this is my first time seeing it)

Reply:
It is a way to return a different status code for the CLI